Dockerfile

```diff
@@ -62,14 +65,14 @@ FROM build AS engine-stage
 
 WORKDIR /engine
 
-## Copy files
-COPY .yarnrc ormconfig.js tsconfig.json ./
-COPY src src
-
 ## Install stage dependencies
 COPY package.json yarn.lock ./
 RUN yarn install
 
+## Copy files
+COPY .yarnrc ormconfig.js tsconfig.json ./
```

Comment:
The `.yarnrc` line, at least, needs to be moved back up where it was, or we will get build issues in the Docker ARM64 cross-compile because of package download timeouts.

Reply:
Hmm I see, so the `.yarnrc` need to exists before `yarn install` is called, right?

Reply:
Yes